riva/client/argparse_utils.py

```diff
@@ -48,6 +48,36 @@ def add_asr_config_argparse_parameters(
         action='store_true',
         help="Flag that controls if speaker diarization should be performed",
     )
+    parser.add_argument(
+        "--start-history",
+        default=0,
+        type=int,
+        help="Value to detect and initiate start of speech utterance",
+    )
+    parser.add_argument(
+        "--start-threshold",
+        default=0.0,
+        type=float,
+        help="Threshold value for detecting the start of speech utterance",
+    )
+    parser.add_argument(
+        "--stop-history",
+        default=0,
+        type=int,
+        help="Value to reset the endpoint detection history",
+    )
+    parser.add_argument(
+        "--stop-history-eou",
+        default=0,
+        type=int,
+        help="Value to determine the response history for endpoint detection",
+    )
+    parser.add_argument(
+        "--stop-threshold",
+        default=0.0,
+        type=float,
+        help="Threshold value for detecting the end of speech utterance",
+    )
```

Comment:
shouldn't the default values of all these fields be same as https://gitlab-master.nvidia.com/dl/riva/riva-speech/-/blob/main/python/servicemaker/servicemaker/modules/asr.py#L558-592

otherwise, it will just set 0 for all these values if these arguments are not specified?

Reply:
Right, the server side is handling <=0 case. But I think we can also add check in this sample client too as well to only send these params to server if value > 0. That will be a reference for users about the valid value that gets honored by server.

Reply:
yeah!! sounds good!!

Reply:
Updated default values